### DISCUSSION TITLE
Bootstrap file is not being shown on after npm install bootstrap@5.3.0-alpha3

### DISCUSSION THREAD

**Discussion by BartBuczek** [twbs/bootstrap#38484] [Q&A]
![image](https://user-images.githubusercontent.com/131296374/233189699-1834bfe4-37c0-4a38-9967-4c0cc31f43f7.png)

Hi, I'm a begginer and I've stumbled across a thing which I can't get over. I've installed the package.json and then the bootstrap dependency as you can see in the zip. My concern is that node_modules won't show bootstrap files even though they are actualy in the project. Could you help me overcome this issue?

[funky (2).zip](https://github.com/twbs/bootstrap/files/11277269/funky.2.zip)

**Comment by julien-deramond** [comment]
Based on the screenshot, your `package.json` seems OK. If I use the same content and run `npm i` at the same level as this `package.json`, I have a `node_modules` directory created containing `@popperjs` (like you) but also `bootstrap` which should be the case.

In your `javascript` directory, you can try to re-run `npm i`.

**Comment by BartBuczek** [comment]
nothing changed unfortunately :(

**Comment by BartBuczek** [comment]
![image](https://user-images.githubusercontent.com/131296374/233194150-2169d7ed-b8a2-4128-a65e-6b0364028484.png)
 node_modules in file explorator contain bootstrap file. However it cannot be seen in the project.
(I've changed the file structure and hoped it would change anything but it didnt)

**Comment by julien-deramond** [comment]
Changed a little bit, your installation with NPM now works correctly since Bootstrap is downloaded and in the `node_modules` directory. Now, you might have an issue while importing the content of Bootstrap in your project. You could check our [Bootstrap examples](https://github.com/twbs/examples) to have something to start from (the [one using Vite for instance](https://github.com/twbs/examples/tree/main/vite) where its content is explained in [our docs](https://getbootstrap.com/docs/5.3/getting-started/vite/) step by step). It will be probably easier if you have no constraints with tools and technologies for your project.

**Comment by BartBuczek** [comment]
thank you, can you tell me then which project fits bootstrap the best? I am curently using "ASP .NET Core - empty" however it contains this tools which you have mentioned.

**Reply by julien-deramond** [reply]
I've transformed the initial issue into a discussion in the Q&A section, not being a bug in Bootstrap but rather in the usage.
TBH I can't really advise for specific tools and technologies to use Bootstrap with, it depends on your background, and what you want to achieve. There's no "better tool" nor "one tool fits it all", so I'd say: if you're used to ASP .NET, great you can start with it.

I wish you a good journey with Bootstrap.

Don't hesitate to use our [Discussions](https://github.com/orgs/twbs/discussions) or [Stack Overflow](https://stackoverflow.com/questions/tagged/bootstrap-5) if you have other questions.
